src/fides/api/schemas/privacy_center_config.py

```diff
@@ -28,16 +51,85 @@ def validate_default_value(cls, values: Dict[str, Any]) -> Dict[str, Any]:
 
 class PrivacyRequestOption(FidesSchema):
     policy_key: str
+    icon_path: str
     title: str
+    description: str
+    description_subtext: Optional[List[str]]
+    confirmButtonText: Optional[str]
+    cancelButtonText: Optional[str]
     identity_inputs: Optional[IdentityInputs] = None
     custom_privacy_request_fields: Optional[Dict[str, CustomPrivacyRequestField]] = None
 
-    class Config:
-        extra = "ignore"
+
+class ConsentConfigButton(FidesSchema):
+    description: str
+    description_subtext: Optional[List[str]]
+    confirmButtonText: Optional[str]
+    cancelButtonText: Optional[str]
+    icon_path: str
+    identity_inputs: IdentityInputs
+    custom_privacy_request_fields: Optional[Dict[str, CustomPrivacyRequestField]] = None
+    title: str
+    modalTitle: Optional[str]
+
+
+class ConditionalValue(FidesSchema):
+    value: bool
+    globalPrivacyControl: bool
+
+
+class ConfigConsentOption(FidesSchema):
+    cookieKeys: List[str] = []
+    default: Optional[Union[bool, ConditionalValue]]
+    description: str
+    fidesDataUseKey: str
+    highlight: Optional[bool]
+    name: str
+    url: str
+    executable: Optional[bool]
+
+
+class ConsentConfigPage(FidesSchema):
+    consentOptions: List[ConfigConsentOption]
+    description: str
+    description_subtext: Optional[List[str]]
+    policy_key: Optional[str]
+    title: str
+
+    @root_validator(pre=True)
+    def validate_consent_options(cls, values: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
again, i think this could be a bit more defensive given that it's a `pre` validator. i realize that the values here may be constrained by the FE client, but better to make as few assumptions as possible about the input values to keep the API robust.